### DISCUSSION TITLE
loosing left side menu when activating tabs

### DISCUSSION THREAD

**Discussion by alinelena** [squidfunk/mkdocs-material#5564] [Q&A] [answered]
I would like to have both tabs(sticky) and left hand side menu active at the same time... unfortunately once I activate tabs I loose the menu on the left hand side.

is this a feature or a bug in my config?
my mkdocs.yml is in here https://gitlab.com/m-lego/m65/-/blob/devel/mkdocs.yml

my understanding is that shall work but somehow I fail

**Comment by squidfunk** [comment, marked as answer]
All working by design. Once you activate tabs, the left hand menu will start at the second layer of navigation, not at the top-level one, as navigation tabs are now dividing your documentation into multiple top level sections. IMHO it makes no sense to deduplicate the entire navigation. This can also be seen on our documentation, e.g. [Reference](https://squidfunk.github.io/mkdocs-material/reference/) only shows the reference section.

**Reply by alinelena** [reply]
thank you indeed.. was my misunderstanding how the system works now is clear.
